### ISSUE TITLE
headers middleware: custom header name not being displayed in dashboard

### ISSUE THREAD

**Issue by Nautigsam** [traefik/traefik#6733]
### Do you want to request a *feature* or report a *bug*?

Bug

### What did you do?

I define a middleware with docker provider:
```yaml
labels:
  - "traefik.http.middlewares.nc-custom-headers.headers.customresponseheaders.Strict-Transport-Security=max-age=15552000"
```

I have activated Dashboard so I open the corresponding middleware page.

### What did you expect to see?

I expect to see both the name and the value of the header I set.

### What did you see instead?

![image](https://user-images.githubusercontent.com/7353930/80302862-eb4c6b00-87ac-11ea-94e0-6c9e2465da79.png)

I only see the value of the header.

### Output of `traefik version`: (_What version of Traefik are you using?_)

```
Version:      2.2.0
Codename:     chevrotin
Go version:   go1.14.1
Built:        2020-03-25T17:32:57Z
OS/Arch:      linux/amd64
```

### What is your environment & configuration (arguments, toml, provider, platform, ...)?

```yaml
# docker provider
labels:
  - "traefik.http.middlewares.nc-custom-headers.headers.customresponseheaders.Strict-Transport-Security=max-age=15552000"
```

**Comment by kevinpollet** [comment]
Hi @Nautigsam and thanks for reporting this bug.

I have been able to reproduce the issue.

**Comment by traefiker** [comment]
Closed by #7706.
